style/nav.css

```diff
@@ -0,0 +1,126 @@
+.navbar {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  font-size: 18px;
+  position: fixed;
+  width: 100%;
+  text-decoration: none;
+  padding: 5px;
+  z-index: 2;
+  background-color: var(--background-color);
+  color: var(--white-color);
+}
+
+.navbar-list {
+  display: flex;
+  list-style: none;
+  text-decoration: none;
+  width: 100%;
+  justify-content: flex-end;
+  gap: 20px;
+  padding-top: 20px;
+  padding-right: 80px;
+  color: var(--white-color);
+}
+.navbar ul li a {
+  color: var(--white-color);
+}
+
+.navbar-list a {
+  text-decoration: none;
+}
+.Logo {
+  padding: 20px 20px 20px 90px;
+  font-size: 30px;
+  text-decoration: none;
+  color: var(--white-color);
+}
+
+.main-section {
+  height: 500px;
+  padding-left: 120px;
+  
+}
+.main-section-container {
+  display: flex;
+  flex-direction: row wrap;
+  justify-content: space-between;
+  align-items: center;
+  
+}
+.left-main-section-container {
+  flex-basis: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin-top: 80px;
```

Comment:
u can make diff. classes for flex... so it will help u not to repeat code.

Reply:
Sure I will do that.Thankyou

Reply:
👍👍👍